app/views/vacancies/index.html.haml

```diff
@@ -33,6 +33,15 @@
             %ul.govuk-list.govuk-list--bullet
               - t('jobs.no_jobs.suggestions').each do |list_item|
                 %li= list_item
+        - if @vacancies_search.wider_radius_suggestions.present?
+          .divider-bottom
+            - if @vacancies_search.keyword.present?
+              .govuk-heading-m= t('jobs.no_jobs.radius_suggestions.heading.keyword_html', keyword: @vacancies_search.keyword)
+            - else
+              .govuk-heading-m= t('jobs.no_jobs.radius_suggestions.heading.no_keyword')
+            %ul.govuk-list.govuk-list--bullet
+              - @vacancies_search.wider_radius_suggestions.each do |wider_radius|
+                %li= link_to("#{wider_radius.first} miles", jobs_path(@vacancies_search.params_hash.merge(radius: wider_radius.first)), class: "govuk-link wider-radius-gtm") + ' ' + t('jobs.no_jobs.radius_suggestions.results', count: wider_radius.last)
```

Comment:
is the gtm html class intentional?

Reply:
i dont see anything about it in acceptance criteria?

Reply:
I thought PA might like it, but I forgot to ask Steven lol

Reply:
It's useful for the time being, so it stays!